automation/terraform/modules/kubernetes/testnet/locals.tf

```diff
@@ -18,6 +18,7 @@ locals {
     logLevel             = var.log_level
     logSnarkWorkGossip   = var.log_snark_work_gossip
     logPrecomputedBlocks = var.log_precomputed_blocks
+    log_txn_pool_gossip = var.log_txn_pool_gossip
```

Comment:
style nit: the lhs of this should be named `logTxnPoolGossip` to match the helm variable naming pattern.

Reply:
Looking below, this name mismatch is why the helm chart isn't working.